modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/tests/RequiredFields.spec.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+
+import {validate} from '../components/EditNotificationTemplate';
+
+const notificationTemplate: NotificationTemplate = {
```

Comment:
Hey @igor-franca, Since this is a mock structure, it would be best to always use the mock suffix `notificationTemplateMock`